src/Compilers/CSharp/Test/Emit/Emit/DeterministicTests.cs

```diff
@@ -373,7 +373,7 @@ static ImmutableArray<INamedTypeSymbol> getForwardedTypes(AssemblySymbol assembl
             }
         }
 
-        [ConditionalFact(typeof(ClrOnly), Reason = "Static execution is runtime defined and this tests Clr behavior only")]
+        [ConditionalFact(typeof(NotOnMonoCore), Reason = "Static execution is runtime defined and this tests Clr behavior only")]
```

Comment:
With `NotOnMonoCore`, the test would get executed on Mono Desktop, right? But it presumably wouldn't work there since it used to be marked `ClrOnly`. Should we change it to `CoreClrOnly` instead?

Reply:
Oh, right, is that because `ExecutionConditionUtil.IsCoreClr` is `true` when running on mono core?

Well, still, we want to skip this test on both MonoCore and MonoDesktop, don't we? Perhaps we should have a `NotOnAnyMono` attribute?

Reply:
Yes you are right . IsCoreClr is just a negation of IsDesktop which just check "RuntimeUtilities.IsDesktopRuntime" which is true when running on mono core. 
yes, We should have NotOnAnyMono attribute. Will make these changes .